js/app.js

```diff
@@ -9,20 +9,42 @@ inp.addEventListener("keyup", () => {
 })
 
 // functions
-getRez()
-
+getRez();
+// replace all null instances with the string "Unknown", null will show as undefined, not a user friendly term/experience;
+function replaceNull(obj) {
```

Comment:
this function will replace all instances of null/undefined with the word "Unknown" by searching for all null keys and replacing their values from null to unknown.